src/screens/UserPortal/Events/Events.tsx

```diff
@@ -76,8 +80,15 @@ export default function events(): JSX.Element {
     },
   });
 
+  const { data: orgData } = useQuery(ORGANIZATIONS_LIST, {
+    variables: { id: organizationId },
+  });
+
   const [create] = useMutation(CREATE_EVENT_MUTATION);
 
+  const userId = localStorage.getItem('id') as string;
+  const userRole = localStorage.getItem('UserType') as string;
```

Comment:
I'm not sure if `UserType` sets in the local storage, when we login directly from the User Portal. Please check. I think in that case, we'll face error as you have used the `as` keyword here.

Reply:
Also check for `id`. Please fix it if it could be `undefined`.

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/97614113/9744a41c-498e-4459-82ae-4759cb12e788)


![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/97614113/e4fc91e0-0fb2-4a45-bb8c-18d8988fbea9)

Yes, we are storing `UserType` and `id` in local storage both in user as well as admin portal.